Alignment/OfflineValidation/README_JetHT

```diff
@@ -0,0 +1,285 @@
+## JetHT validation
```

Comment:
please add the file extension .md such that it gets formatted by gitHub.

Reply:
ok